src/Commands/ConstructCommand.php

```diff
@@ -54,15 +63,17 @@ class ConstructCommand extends Command
     /**
      * Initialize.
      *
-     * @param \JonathanTorres\Construct\Construct $construct
-     * @param \JonathanTorres\Construct\Str       $str
+     * @param \JonathanTorres\Construct\Construct  $construct
+     * @param \JonathanTorres\Construct\Str        $str
+     * @param \JonathanTorres\Construct\Filesystem $filesystem
```

Comment:
This doesn't match the argument. Typehint is pointing to ` JonathanTorres\Construct\Helpers\Filesystem` instead of `JonathanTorres\Construct\Filesystem`

Also, leading backslashes aren't needed.